### DISCUSSION TITLE
ReTux levelsets

### DISCUSSION THREAD

**Discussion by ScriptForAll** [retux-game/retux#10] [Q&A] [answered]
Hi!
I know this isn't the channel to ask this is there a repository of levelsets for this game similiar to what SuperTux has?

**Comment by onpon4** [comment, marked as answer]
There isn't. So far as I'm aware, the only levelsets that have ever been made are the ones included with the game. (Incidentally, if anyone were to develop additional reasonably high-quality levelsets for ReTux, we would be more inclined to just bundle them with ReTux, as well, rather than maintain a separate repo for them.)

For what it's worth, I too would be interested in seeing others make levels (that's why I wrote the [Level Editing Guide](https://retux-game.github.io/leveledit/)).

EDIT: Something of a side-note, you posting this made me realize that I never enabled the "Discussions" feature on this repo, so I've remedied that and converted this into a discussion.

🦇

**Comment by ScriptForAll** [comment]
Uh~ I will check that guide then, seems interesting.

Regarding the fact that there aren't many levelsets for this game: have you made any promotion to the game to attract more people into the game?
Places like the SuperTux forums or the Supertux github could be a starting point.

I love the take that this game has on SuperTux and would like to see more people to give it a try :smile:

**Comment by onpon4** [comment]
I don't remember what we did exactly to promote the game when we first pushed it out years ago, but probably posted it to a couple of forums we were on at the time, and I do remember that we've posted about recent releases on Reddit.

I don't think it would be appropriate for us to post about ReTux in SuperTux's community since we aren't a part of it, and I doubt the SuperTux maintainers would appreciate us using their community solely to promote a different game. If you want more people to give it a try (and I agree), all I can say is, spread it around in the circles you're a part of. We've always exhaustively used the avenues we have to tell people about the game, but there's only so much one system can do (even moreso now that we struggle with long COVID).

🦇

**Comment by jorgesumle** [comment]
I've just created a trailer of the game and uploaded it to both PeerTube and YouTube:
- https://video.hardlimit.com/w/5FBgTf6B5ccCcBcHxdJgjW (PeerTube)
- https://yewtu.be/watch?v=z65OLwsol5I (YouTube)

@onpon4, I noticed that the trailer that you uploaded years ago to YouTube has a worse resolution (360p) than the actual game, which is 800x480. If you don't want the resolution to be reduced by YouTube after the upload, you have to resize the video to 720p before uploading it. I don't like it, but that's how YouTube works. I think that both the official website and videos are very important to advertise the game. I would like to translate the official website into Spanish, so that more Spanish speakers can find the game.

**Reply by onpon4** [reply]
Thanks!

Truthfully, I've been unhappy with that old trailer for a rather long while. The reason it's only 360p is because the original size of that recording was 800×448, which isn't quite 480p. ReTux's resolution is 800×480 now, so a new recording wouldn't face the same issue I don't think.

In any case, I largely haven't bothered to make a new ReTux trailer because I honestly don't have a lot of passion for ReTux anymore, compared to Naikari, Keeper of the Cards, or even Hexoshi (which, despite being on hiatus right now, I do very much want to get back to working on eventually). Besides, I don't think ReTux will ever cease to live in the shadow of SuperTux, partly because it doesn't do anything particularly distinct.

If you'd like to translate the website, the best way I can think of to do that would be to make copies of the HTML files in language-specific subdirectories. It's at https://github.com/retux-game/retux-game.github.io if you wanna do that.

**Reply by jorgesumle** [reply]
> I don't think ReTux will ever cease to live in the shadow of SuperTux, partly because it doesn't do anything particularly distinct.

I really like the vision of ReTux. Tux dolls, the ability to throw fire both upwards and downwards, the way objects are picked up automatically... They are very similar games, but ReTux has some nice features. I really enjoy playing this game. SuperTux is also a great game.

> Naikari

I played Naev, but not much. Maybe I like Naikari more. I'll give the game a try when I have more free time.

> Hexoshi (which, despite being on hiatus right now, I do very much want to get back to working on eventually)

That's great!

> It's at https://github.com/retux-game/retux-game.github.io if you wanna do that.

I will do that when I have some free time. I probably won't bother translating the "Level Editing Guide".
